wdl/transforms/draft3/src/test/cases/empty_call_workflow.nowom.wdl

```diff
@@ -0,0 +1,10 @@
+version draft-3
+
+workflow empty_call {
```

Comment:
I'm not a huge fan of the name `empty_call`... I think `no_input_no_output` is more precise although less snappy...